public/js/home.js

```diff
@@ -52,6 +52,54 @@ window.addEventListener("click", function(event) {
         tableData_closeAllSelect();
     }
 });
+// detect color scheme and set slider
+window.addEventListener("load", loadMode);
+function loadMode() {
+    // loads current os mode and sets slider accordingly
+    const slider = document.getElementById("dark-check");
+    let currentMode = "";
+    if (window.matchMedia("(prefers-color-scheme: dark)").matches) {
+        document.body.classList.toggle("dark");
+        currentMode = "dark";
+        // sets the slider to match the actual mode
+        slider.checked = true;
+    }
+    else {
```

Comment:
`else` should be on the same line as the closing brace
```suggestion
    } else {
```